setup.py

```diff
@@ -60,17 +68,21 @@ def read_version():
     'Programming Language :: Python :: 3.6',
     'Programming Language :: Python :: 3.7',
     'Operating System :: POSIX',
-    'Development Status :: 4 - Beta'
+    'Development Status :: 4 - Beta',
     'Framework :: AsyncIO',
 ]
 
 
 setup(
     name="create-aio-app",
     version=read_version(),
-    description=("The tool that helps quickly create a basis for "
-                 "the microservice on aiohttp and prepare the development "
-                 "environment."),
+    description=(
+        "The tool that helps quickly create a basis for "
+        "the microservice on aiohttp and prepare the development "
+        "environment."
+    ),
+    long_description=get_text_from('README.md'),
```

Comment:
If you switch to rst, you'll be able to embed (parts of) readme into sphinx docs

Reply:
\+